datasets/dataset_setup.py

```diff
@@ -367,15 +368,29 @@ def setup_imagenet_jax(data_dir):
 
   # Setup jax dataset dir
   imagenet_jax_data_dir = os.path.join(data_dir, 'jax')
-  os.makedirs(imagenet_jax_data_dir)
-
-  # Copy tar file into jax
-  logging.info('Copying {} to {}'.format(train_tar_file_path,
-                                         imagenet_jax_data_dir))
-  shutil.copy(train_tar_file_path, imagenet_jax_data_dir)
-  logging.info('Copying {} to {}'.format(val_tar_file_path,
-                                         imagenet_jax_data_dir))
-  shutil.copy(val_tar_file_path, imagenet_jax_data_dir)
+  manual_download_dir = os.path.join(imagenet_jax_data_dir,
+                                     'downloads',
+                                     'manual')
+  os.makedirs(manual_download_dir, exist_ok=True)
+
+  # Copy tar file into jax/downloads/manual
+  logging.info('Checking if tar files already exists in jax/downloads/manual.')
+  if not os.path.exists(
+      os.path.join(manual_download_dir, IMAGENET_TRAIN_TAR_FILENAME)):
+    logging.info('Copying {} to {}'.format(train_tar_file_path,
+                                           manual_download_dir))
+    shutil.copy(train_tar_file_path, manual_download_dir)
+  if not os.path.exists(
+      os.path.join(manual_download_dir, IMAGENET_VAL_TAR_FILENAME)):
+    logging.info('Copying {} to {}'.format(val_tar_file_path,
+                                           manual_download_dir))
+    shutil.copy(val_tar_file_path, manual_download_dir)
```

Comment:
can we avoid having to do these copies? they could take up a lot of disk space (at least the train tar) if we have 2 copies